teachertool/src/components/CriteriaInstanceDisplay.tsx

```diff
@@ -158,7 +157,10 @@ export const CriteriaInstanceDisplay: React.FC<CriteriaInstanceDisplayProps> = (
                     </span>
                 ))}
             </div>
-            <div className={css["criteria-description"]}>{catalogCriteria.description}</div>
+            <div className={classList(css["criteria-description"], "no-print")}>{catalogCriteria.description}</div>
```

Comment:
For the criteria-description, is the only difference between the two the italics? If so, could that be done with a single control inside a `@media print` section of the css?